src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -467,6 +467,20 @@ async function createEmscriptenModuleInstance(resourceLoader: WebAssemblyResourc
       // -1 enables debugging with logging disabled. 0 disables debugging entirely.
       MONO.mono_wasm_load_runtime(appBinDirName, hasDebuggingEnabled() ? -1 : 0);
       MONO.mono_wasm_runtime_ready();
+      try {
+        BINDING.bind_static_method('invalid-fqn', '');
+      } catch (e) {
+        // HOTFIX: until https://github.com/dotnet/runtime/pull/72275
+        // this would always throw, but it will initialize runtime interop as side-effect
+      }
+
+      // makes Blazor._internal visible to [JSImport]
+      Object.assign(IMPORTS, {
+        Blazor: {
+          _internal: Blazor._internal,
+        },
+      });
```

Comment:
If your users override any emscripten events and add something to IMPORTS object before this call, we want to merge. It's not deep merge, you own `Blazor` so that's fine. And it's the instance of `internal` which will get further updates.

Reply:
I didn't want to make all Blazor accessible.

Reply:
From what you're saying, it sounds as if the `Object.assign` call is functionally identical to just doing a regular assignment. If it's not, could you clarify how it differs from doing this instead of lines 478-482?

```ts
IMPORTS.Blazor = { _internal: Blazor._internal };
```